lib/mvi/service.rb

```diff
@@ -141,29 +141,24 @@ def build_exception(key, error)
       )
     end
 
-    def mvi_error_handler(user_identity, e)
+    def mvi_error_handler(user_identity, e, source = '')
       case e
       when MVI::Errors::DuplicateRecords
-        log_console_and_sentry('MVI Duplicate Record', :warn)
+        log_message_to_sentry('MVI Duplicate Record', :warn)
       when MVI::Errors::RecordNotFound
-        log_console_and_sentry('MVI Record Not Found')
+        Rails.logger.info('MVI Record Not Found')
```

Comment:
Just double-checking that it was intentional to use `Rails.logger` here, without also calling `log_message_to_sentry`?

Also, I don't see a spec example for the `Rails.logger` use-case. Is that something you want to include?

Reply:
yeah, that's intentional.  `log_console_and_sentry` was just writing to the log unless there was a second argument.  And, this error is "we didn't find you in MVI" which is expected to happen often and not really an error.